specs/language/statements.tex

```diff
@@ -1,2 +1,74 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
```

Comment:
visual of : [b161303](https://github.com/microsoft/hlsl-specs/pull/263/commits/b1613031a53abf2f1ac0d2052257b617aa98b6f7)
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/00666515-df91-4557-a3b5-fe6c1195c61c)
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/f6f0e993-624e-481d-b632-85db8fd291bf)

Reply:
visual of  https://github.com/microsoft/hlsl-specs/pull/263/commits/3d4f3984157a6f655986b3ebe3b11cbd118a938f
unroll spec:
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/5eb4455e-ca7e-42f3-97f2-898baf416aea)

loop spec:
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/9225bfd3-f631-4d8d-9ee7-e3611bf2d080)

Reply:
visual of  https://github.com/microsoft/hlsl-specs/pull/263/commits/aec93914bb9a77ed8759e6244929fb3e18ab95f2
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/812fbfe3-0adb-4477-81dc-1b3444cedfd1)